changedetectionio/forms.py

```diff
@@ -529,7 +529,7 @@ def validate(self, **kwargs):
         except ModuleNotFoundError as e:
             # incase jinja2_time or others is missing
             logger.error(e)
-            self.headers.errors.append(f'Invalid template syntax configuration: {e}')
+            self.url.errors.append(f'Invalid template syntax configuration: {e}')
```

Comment:
I wondered about that too since I didn't see a pattern like this used elsewhere. It appears the underlying classes protect against this. E.g. --

<img width="681" alt="image" src="https://github.com/user-attachments/assets/ac6da75e-eb24-437f-a933-bcc2baf7c8a6">

<img width="863" alt="image" src="https://github.com/user-attachments/assets/7884ae73-b818-4b61-8d0f-fd86aa4e02bc">